core/nodejs14MultitenantAction/supervisord.conf

```diff
@@ -0,0 +1,12 @@
+[supervisord]
+nodaemon=true
+
+[program:nodeServer]
+directory=/nodejsAction
+command=node --expose-gc app.js 
+autostart=true
+autorestart=true
+user=root
+startsecs=10
+stdout_logfile=/var/log/supervisor/node.log
```

Comment:
this should go to `/dev/stdout` same for `/dev/stderr`

Reply:
fixed